test/bats/tests/config/vulnerabilityreport_config.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Not for this PR, wondering are we going to wire this verifier up into default chart?

Reply:
discussed in community meeting that follow up PR will wire this up